tools/build-tools/src/bumpVersion/bumpVersionCli.ts

```diff
@@ -26,6 +26,7 @@ Options:
   -r --release [<pkg>[=<type>]]  Release and bump version of specified package or monorepo and dependencies (default: client)
   -u --update                    Update prerelease dependencies for released packages
      --version [<pkg>[=<type>]]  Collect and show version of specified package or monorepo and dependencies (default: client)
+     --virtualPatch              Use a virtual patch number for beta versioning (0.<major>.<minor>00<patch>)
```

Comment:
you have a pretty good description of this in the pr, should that also go in the readme?

Reply:
sure i'll add it there, just forgot that readme existed